homeassistant/components/tesla/__init__.py

```diff
@@ -202,6 +218,17 @@ async def async_unload_entry(hass, config_entry) -> bool:
     return False
 
 
+def _async_start_reauth(hass: HomeAssistant, entry: ConfigEntry):
+    hass.async_create_task(
+        hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": "reauth"},
+            data=entry.data,
+        )
+    )
+    _LOGGER.error("Credentials are is no longer valid. Please reauthenticate")
```

Comment:
Typo: 
Should be:

Reply:
`Credentials are no longer valid. Please reauthenticate`

Reply:
Good catch. Thanks!